packages/p2p/src/stores/advertiser-page-store.js

```diff
@@ -1,16 +1,13 @@
 import { action, computed, makeObservable, observable } from 'mobx';
 import { buy_sell } from 'Constants/buy-sell';
-import { getShortNickname } from 'Utils/string';
-import { requestWS } from 'Utils/websocket';
+import { requestWS, subscribeWS } from 'Utils/websocket';
 import BaseStore from 'Stores/base_store';
 
 export default class AdvertiserPageStore extends BaseStore {
     active_index = 0;
     ad = null;
-    advertiser_first_name = '';
-    advertiser_last_name = '';
-    advertiser_info = {};
```

Comment:
can we also remove these from the constructor (lines 29 - 31 and 60-62)

Reply:
Yeap, a lot of it got lost in the conflicts.